config/default.json

```diff
@@ -1,5 +1,5 @@
 {
-  "database_url": "sqlite:///tmp/sqlite.db",
+  "database_url": "mysql://root:password@127.0.0.1/steemit_dev",
```

Comment:
Up to you guys - I am in favor of allowing condenser to continue to work on a local dev instance very simply, but I know with the signup changes we've made as well as the forthcoming dependency on `steemit/hivemind` that that might be unrealistic.  Your call.